cmd/clusterctl/client/alpha/rollout_restarter.go

```diff
@@ -53,3 +65,8 @@ func setRestartedAtAnnotation(proxy cluster.Proxy, name, namespace string) error
 	patch := client.RawPatch(types.MergePatchType, []byte(fmt.Sprintf("{\"spec\":{\"template\":{\"metadata\":{\"annotations\":{\"cluster.x-k8s.io/restartedAt\":\"%v\"}}}}}", time.Now().Format(time.RFC3339))))
 	return patchMachineDeployment(proxy, name, namespace, patch)
 }
+
+func setRolloutAfter(proxy cluster.Proxy, name, namespace string) error {
+	patch := client.RawPatch(types.MergePatchType, []byte(fmt.Sprintf("{\"spec\":{\"rolloutAfter\":\"%v\"}}", time.Now().Format(time.RFC3339))))
```

Comment:
Another idea to handle issues regarding different time zones or async local times; we could use the creation timestamp + a few seconds. In this case the rolling update should start in any case

Reply:
And btw, e.g. `Deployment` has also a `restartedAt` field like our `MachineDeployment`.

```
❯ kk rollout restart deploy coredns
deployment.apps/coredns restarted
❯ kk get deploy coredns -oyaml | grep restartedAt
        kubectl.kubernetes.io/restartedAt: "2022-07-18T12:09:53+02:00"
```

Same for `Statefulsets`

```
❯ kk rollout restart sts csi-resizer-cinder
statefulset.apps/csi-resizer-cinder restarted
❯ kk get sts csi-resizer-cinder -oyaml | grep restartedAt
        kubectl.kubernetes.io/restartedAt: "2022-07-18T12:11:06+02:00"
```

Reply:
Making use of `rolloutAfter` would limit the functionality for RolloutAfter.

E.g. a user may have already set `RolloutAfter` to a timestamp in the future (e.g. to enforce recreation of the control plane to renew certificates).

So what would happen if `RolloutAfter` is already set (and because of that we should not overwrite it?)